plugins/chat/app/controllers/chat/chat_controller.rb

```diff
@@ -412,12 +418,22 @@ def preloaded_chat_message_query
           .includes(:bookmarks)
           .includes(:uploads)
           .includes(chat_channel: :chatable)
+          .includes(:thread)
 
       query = query.includes(user: :user_status) if SiteSetting.enable_user_status
 
       query
     end
 
+    def echo_thread_messages
```

Comment:
Not sure it's worth even adding this for now without a UI toggle, will remove if not

Reply:
Yes lets remove it, carrying stuff we might never use has more risks than advantages IMO